roles/node_images/files/scripts/common/create-ims-initrd.sh

```diff
@@ -37,9 +37,19 @@ dracut \
 
 echo "Copying vmlinuz and initrd into /squashfs for disk-bootloader setup."
 rm -f /squashfs/*
-cp -pv /boot/vmlinuz-${KVER} /squashfs/${KVER}.kernel
 cp -pv /boot/initrd-${KVER} /squashfs/initrd.img.xz
 
+if [[ -f /boot/vmlinuz-${KVER} ]]; then
+  # x86_64 kernel
+  cp -pv /boot/vmlinuz-${KVER} /squashfs/${KVER}.kernel
+elif [[ -f /boot/vmlinux-${KVER}.gz ]]; then
+    # aarch64 kernel
+  cp -pv /boot/vmlinux-${KVER}.gz /squashfs/${KVER}.kernel
+else
+    echo "ERROR: No kernel found for version ${KVER}."
```

Comment:
Should print to stderr.
```suggestion
    echo >&2 "ERROR: No kernel found for version ${KVER}."
```